src/js/angular/rest/rdf4j.repositories.rest.service.js

```diff
@@ -115,4 +116,28 @@ function RDF4JRepositoriesRestService($http, $repositories, $translate) {
         }
         return graphsInRepo;
     }
+
+    function downloadResultsAsFile(repositoryId, queryParams, acceptHeader) {
+        return $http.get(`${REPOSITORIES_ENDPOINT}/${repositoryId}`, {
+            headers: {
+                accept: acceptHeader
+            },
+            params: queryParams,
+            responseType: "blob"
+        }).then(function (res) {
```

Comment:
please add a catch clause with a proper toastr message

Reply:
There is a catch where it is used. It does not enter here if there is an error.